product_docs/docs/pem/10/debugger.mdx

```diff
@@ -1,136 +1,129 @@
 ---
-title: "PL Debugger"
-navTitle: "Using PL Debugger"
+title: "PL debugger"
+navTitle: "Using the PL debugger"
 ---
```

Comment:
This doesn't seem to be a product name requiring upper case, and it would also need an article in the nav if not a product name.